src/completion.ts

```diff
@@ -318,17 +319,20 @@ class DictionaryCompletionItemProvider implements vscode.CompletionItemProvider
                 return [];
             // TMP adapted from JS/TS
             case "c":
+            case "verilog":
                 //// Multiline comment
                 if (/\/\*((?!\*\/)[\W\w])*$/.test(docTextBefore)) {
                     return this.completeByFirstLetter(firstLetter, addSpace);
                 }
                 //// Inline comment or string
                 const tmpTextBeforeC = textBefore.replace(/(?<!\\)(").*?(?<!\\)\1/g, '');
+                // C, verilog use # and `(backtick) as preprocess directive respectively
+                const includeStr = this.fileType == "c" ? /#include/ : /`include/;
```

Comment:
I'd prefer a name that indicates it is a regex pattern, e.g. `includeStrPat(tern)`?

Reply:
Have renamed it.